### DISCUSSION TITLE
Any way to access the main stream for a still image?

### DISCUSSION THREAD

**Discussion by gururise** [blakeblackshear/frigate#3623] [General]
Once every 5 minutes or so, I would like to be able to pull 1 still frame off the main stream for image analysis on a remote server.  Right now, I use the http api endpoint: **GET /api/_cam_name_/latest.jpg** 

Unfortunately, that only provides a still image from the detect stream, which is a low-res substream.  Is there any way to access the main stream and either:

1. grab the raw rgb data for one frame (I can encode to jpg on the remote server)
or 
2. grab one jpeg encoded frame from the main stream

**Comment by NickM-27** [comment]
Pulling a frame from a stream requires decoding the stream, frigate doesn't support decoding the record stream currently
